compiler-core/src/parse/snapshots/gleam_core__parse__tests__list_spread_followed_by_another_spread.snap

```diff
@@ -0,0 +1,23 @@
+---
+source: compiler-core/src/parse/tests.rs
+expression: "\npub fn main() -> Nil {\n  let xs = [1, 2, 3]\n  let ys = [5, 6, 7]\n  [..xs, 4, ..ys]\n}\n"
+---
+----- SOURCE CODE
+
+pub fn main() -> Nil {
+  let xs = [1, 2, 3]
+  let ys = [5, 6, 7]
+  [..xs, 4, ..ys]
+}
+
+
+----- ERROR
+error: Syntax error
+  ┌─ /src/parse/error.gleam:5:4
+  │
+5 │   [..xs, 4, ..ys]
+  │    ^^ I wasn't expecting a spread here
```

Comment:
In this case the new error shouldn't be returned, instead it should be the existing error for there being elements after a spread. This error should only happen when there's 2 spreads but no elements between them.